pkg/fluentd/fluentd_test.go

```diff
@@ -90,6 +91,45 @@ func TestCreateFluentdContainer(t *testing.T) {
 	assertion.Equal("/u01/data/docker/containers", fluentd.VolumeMounts[2].MountPath, "Fluentd container volume mounts mount path not equal to expected value")
 	assertion.Equal("datadockercontainers", fluentd.VolumeMounts[2].Name, "Fluentd container volume mounts name not equal to expected value")
 	assertion.Equal(true, fluentd.VolumeMounts[2].ReadOnly, "Fluentd container volume mounts read only flag not equal to expected value")
+
+	// In addition test that when Dev mode is set we get the expected elastic search host
+	os.Setenv("SINGLE_SYSTEM_VMI", "true")
+	fluentd = CreateFluentdContainer("test-binding", "test-component")
+	os.Unsetenv("SINGLE_SYSTEM_VMI")
+
+	assertion.Equal("fluentd", fluentd.Name, "Fluentd container name not equal to expected value")
+	assertion.Equal(2, len(fluentd.Args), "Fluentd container args count not equal to expected value")
+	assertion.Equal("-c", fluentd.Args[0], "Fluentd container arg not equal to expected value")
+	assertion.Equal("/etc/fluent.conf", fluentd.Args[1], "Fluentd container arg not equal to expected value")
+	assertion.Equal(7, len(fluentd.Env), "Fluentd container envs count not equal to expected value")
```

Comment:
Maybe should fail if len is wrong to avoid potential out of bounds condition in subsequent checks.

Reply:
same as above